pkg/network/netbinding/netbinding_test.go

```diff
@@ -157,6 +157,34 @@ var _ = Describe("Network Binding", func() {
 			}}))
 		})
 
+		It("should create a slirp and a custom hook sidecar", func() {
+			fakeRecorder := record.NewFakeRecorder(1)
+			config := &v1.KubeVirtConfiguration{
+				ImagePullPolicy: k8scorev1.PullIfNotPresent,
+				NetworkConfiguration: &v1.NetworkConfiguration{
+					Binding: map[string]v1.InterfaceBindingPlugin{
+						testBindingName1: {SidecarImage: testSidecarImage1},
+						"slirp":          {SidecarImage: "kubevirt/network-slirp-plugin"},
+					},
+				},
+			}
+
+			vmi.Spec.Domain.Devices.Interfaces = append(vmi.Spec.Domain.Devices.Interfaces, v1.Interface{
+				Name:    "testnet2",
+				Binding: &v1.PluginBinding{Name: testBindingName1},
+			})
+			Expect(netbinding.NetBindingPluginSidecarList(vmi, config, fakeRecorder)).To(ConsistOf(hooks.HookSidecarList{
+				{
+					ImagePullPolicy: k8scorev1.PullIfNotPresent,
+					Image:           "kubevirt/network-slirp-plugin",
+				},
+				{
+					ImagePullPolicy: k8scorev1.PullIfNotPresent,
+					Image:           testSidecarImage1,
```

Comment:
nit: There is a mix and match of defined variables and raw strings, e.g.: "testnet2" (line 173) and `testBindingName1` (line 174).
Using the already defined variables instead (line 45-46, `testNetworkName2` ` testBindingName2`  `testSidecarImage2`) or similar raw strings (e.g: "testBinding2"..) would be nicer and less confusing.

Reply:
Well, I can only guess that you wanted me to use `testNetworkName1` instead of `testnet2` at line 173.

Done